analyzer/tests/functional/analyze/test_analyze.py

```diff
@@ -1065,7 +1101,7 @@ def test_analyzer_and_checker_config(self):
             cwd=self.test_dir,
             encoding="utf-8",
             errors="ignore")
-        out, _ = process.communicate()
+        out, err = process.communicate()
```

Comment:
```suggestion
        out, _ = process.communicate()
```
PyLint is complaining about this unused variable, otherwise it looks good.